package-lock.json

```diff
@@ -0,0 +1,3871 @@
+{
```

Comment:
This shouldn't be there, the only lockfile should be `pnpm-lock.yaml`. Did you install something using `npm` instead of `pnpm`?